webknossos/webknossos/annotation/annotation.py

```diff
@@ -124,6 +146,63 @@ def download(cls, annotation_path: str) -> "Annotation":
 
         return download_annotation(annotation_type, annotation_id)
 
+    def _skeleton_zip_path(self) -> Union[Openable, PathLike, str]:
+        nml_files = [i for i in self._filelist if i.endswith(".nml")]
+        assert len(nml_files) == 1
+        return _ZipPath(self._zipfile, nml_files[0])
+
+    @contextmanager
+    def _open_nml(self) -> Iterator[IO[bytes]]:
+        """
+        This method can be used to open the annotation's inner NML
+        file directly.
+        """
+        file_path = self._skeleton_zip_path()
+
+        if isinstance(file_path, Openable):
+            with file_path.open(mode="rb") as file_handle:
+                yield file_handle
+        else:
+            with open(file_path, "rb") as file_handle:
+                yield file_handle
+
+    def save_to_file(self, path: Union[Path, str]) -> None:
+        """
+        Stores the annotation as a zip at the given path.
+        """
+
+        assert Path(path).suffix == ".zip", "The target path should be a zip file."
+
+        if isinstance(self.file, str) or isinstance(self.file, PathLike):
+            copyfile(self.file, str(path))
+        else:
+            with open(path, "wb") as f:
+                f.write(self.file.getbuffer())
+
+    @contextmanager
+    def temporary_annotation_view(self) -> Iterator[Layer]:
```

Comment:
Nice! I'd maybe rename it to `temporary_volume_annotation_layer` or `temporary_volume_annotation_layer_copy`. (`volume_annotation` for clarity, and I'd rather not use `view`, since it's a copy and writes on the returned object won't be reflected in the annotation.